dts/arm/st/h7/stm32h7b0.dtsi

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <st/h7/stm32h7b3.dtsi>
```

Comment:
No, please get stm32h7b3.dtsi including stm32h7b0.dtsi.

Reply:
Sorry, I didn't get the suggestion.

Reply:
I'd like to have `#include stm32h7b0.dtsi` in stm32h7b3.dtsi. Not the other way round

Reply:
Done. Thanks you!